docs/doc/13-monitor/prometheus-and-granafa.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Prometheus & Grafana
+---
+
+[Prometheus](https://prometheus.io/) is a time series database that features a flexible query language and a multi-dimensional data model.
+
+[Grafana](https://grafana.com/grafana) is an open-source tool used for analyzing and visualizing metrics.
+
+Databend supports using Prometheus to store monitoring and performance metrics, which can be visualized through Grafana.
+
+## Tutorial: Monitor Databend with Prometheus & Grafana
+
+### Step 1. Deploy Databend
+
+Follow the [Deployment Guide](https://databend.rs/doc/deploy) to deploy Databend.
+
+In this example, we use the default configuration located in the `configs` directory of release to deploy a standalone  Databend. You can also find these configurations at [datafuselabs/databend - configs](https://github.com/datafuselabs/databend/tree/main/scripts/distribution/configs).
```

Comment:
:::tip
This tutorial uses the [default configuration files](https://github.com/datafuselabs/databend/tree/main/scripts/distribution/configs) in the `configs` folder of the install package. The metrics API for databend-meta is `0.0.0.0:28101/v1/metrics`, and the metrics API for databend-query is `0.0.0.0:7070/metrics`.
:::